src/sentry/static/sentry/app/components/banner.tsx

```diff
@@ -0,0 +1,133 @@
+import React from 'react';
+import InlineSvg from 'app/components/inlineSvg';
+
+import styled from 'react-emotion';
+import theme from 'app/utils/theme';
+import space from 'app/styles/space';
+import {t} from 'app/locale';
+import localStorage from 'app/utils/localStorage';
+
+import spaceBg from '../../images/background-space.svg';
+
+type Props = {
+  title?: string;
+  subtitle?: string;
+  dismissable?: boolean;
+};
+
+class Banner extends React.Component<Props> {
+  static defaultProps: Partial<Props> = {
+    dismissable: true,
+  };
+
+  handleClick() {
+    localStorage.setItem('banner-dismissed', true);
+  }
+
+  render() {
+    const {title, subtitle, dismissable, children} = this.props;
+    const bannerDismissed = localStorage.getItem('banner-dismissed');
```

Comment:
It may be a good idea to namespace the localstorage key, for example `discoverv2-banner-dismissed`. 

In case we may have another dismissable banner elsewhere.

Reply:
Done